go/chat/boxer.go

```diff
@@ -400,8 +403,9 @@ func (b *Boxer) unbox(ctx context.Context, boxed chat1.MessageBoxed,
 		}
 		return res, err
 	// V3 is the same as V2, except that it indicates exploding message support.
-	case chat1.MessageBoxedVersion_V2, chat1.MessageBoxedVersion_V3:
-		res, err := b.unboxV2orV3(ctx, boxed, membersType, encryptionKey, ephemeralSeed)
+	// V4 is the same as V3, except if pairwise MACs are included, then the sender signing key is a dummy.
+	case chat1.MessageBoxedVersion_V2, chat1.MessageBoxedVersion_V3, chat1.MessageBoxedVersion_V4:
```

Comment:
can you make a ticket for busting the msg storage cache on these version bumps? Right now old clients will store the new version as a permanent error. if we run across an error of this type in storage and we now understand the new version we should try to unbox the message. cc @mmaxim

Reply:
https://keybase.atlassian.net/browse/CORE-8066